contrib/linux-systemd-service/clipper.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Clipper ~ Clipboard proxy
+
+[Service]
+ExecStart=/home/nelson/bin/clipper
```

Comment:
The ExecStart path must be absolute, don't support variable expansion or directory expansion (support it for the command arguments).  
Can be uses something like this too 

    ExecStart=/bin/sh -c '~/bin/clipper'

But that will left sh on memory too  

I updated the script to use a standard directory for the example.